src/components/transactions/modals/ClaimWRBTCModal.vue

```diff
@@ -159,12 +169,26 @@ export default {
           break
         }
         case this.claimTypes.CONVERT_TO_RBTC: {
-          const weiAmount = new BigNumber(this.receiveAmount).shiftedBy(-8).toString()
+          this.processing = true
+          
+          const weiAmount = new BigNumber(this.receiveAmount).shiftedBy(8).toFixed(0)
+
           this.responseEstimatedGas = await this.getEstimatedGasPrice(weiAmount)
-          const estimatedGas = new BigNumber(this.responseEstimatedGas?.amountEstimatedGas)
+          // TODO: check if response is not null otherwise show an error
+          
+          const estimatedGas = new BigNumber(this.responseEstimatedGas?.amount)
             .shiftedBy(-8)
```

Comment:
same here

Reply:
added the issue https://rsklabs.atlassian.net/browse/TBRE-516